views/recent.ejs

```diff
@@ -2,21 +2,21 @@
 <h1>Recently scanned QR codes</h1>
 <article class="col-xs-12">
   <ul class='recent-list'>
-    <% for (var i = 0; i < allScans.length; i++) { %>
+    <% for (let item of allScans.values()) { %>
       <li>
```

Comment:
@mpsido nice! Are there other areas in the code where this rewrite should be applied?

Reply:
Everywhere there is a for loop ;) if you don't care about the index then you can use for..in or for..of.
See the MSDN documentation to understand the difference ;)

Reply:
I mean do it.